R/yoda_test_linter.R

```diff
@@ -45,8 +45,11 @@ yoda_test_linter <- function() {
     or (STR_CONST and not(OP-DOLLAR))
     or ((OP-PLUS or OP-MINUS) and count(expr[NUM_CONST]) = 2)
   "
+  expectations <- c("expect_equal", "expect_identical", "expect_setequal")
+  expectations_xpath <- xp_text_in_table(expectations)
+
   xpath <- glue::glue("
-  //SYMBOL_FUNCTION_CALL[text() = 'expect_equal' or text() = 'expect_identical' or text() = 'expect_setequal']
+  //SYMBOL_FUNCTION_CALL[ { expectations_xpath } ]
```

Comment:
i don't agree this is clearer. `expectations` is too vague and makes us read elsewhere to understand.

Whereas the current version is very clear, since all of the relevant functions are right in front of us.

So I would revert